### ISSUE TITLE
Orange pit itemvar for goo pit

### ISSUE THREAD

**Issue by Nikita74RU** [BEEmod/BEE2.4#896]
Just a orange pit like seen in Portal Pro, at the ending secquence you could see a large orange pit. Will orangifiy your screen, and kill you if you jump in. Works in all styles, basically a background of panels on P1 style, orange glowing pit. Emits orange lights.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
Screenshot please?

**Comment by Nikita74RU** [comment]
No, can't find it. When found, will be shown.

**Comment by Nikita74RU** [comment]
Just a Warm hole light from P1 but its not a hole, its a PIT.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
Like this?
![capture 10](https://user-images.githubusercontent.com/31172185/38771425-1843fd0e-3fd8-11e8-8538-4be5552a5896.PNG)
![capture 9](https://user-images.githubusercontent.com/31172185/38771427-1cc30d0c-3fd8-11e8-8502-85f34ae8fb8d.PNG)

**Comment by Nikita74RU** [comment]
Yes! Broken panels at pits borders will be also included! We need that!

**Comment by Nikita74RU** [comment]
If this pit is present in the map, the whole chamber will glow with orange.
